src/ctia/stores/es/mapping.clj

```diff
@@ -9,59 +9,44 @@
 ;; even in different entities.
 
 (def float-type
-  {:type "float"
-   :include_in_all false})
```

Comment:
The removal of `include_in_all` is mandatory for ES7 init.
In ES7, the default behavior is to search in every fields if none is specified, but unlinke the former `_all` (removed in ES7) this search preserve each type and analyser which offers more precise search.
A deeper analysis of this impact will be covered when tackling https://github.com/threatgrid/iroh/issues/4288.
In any case, this change will not affect existing indices and I disable in this PR the mapping update which would failed due to this change.